### ISSUE TITLE
Integrate mssql-cli intellisense features

### ISSUE THREAD

**Issue by anthonydresser** [microsoft/azuredatastudio#2982]
- Focus on intellisense on azure

Hierarchical intellisense

**Comment by yualan** [comment]
@anthonydresser are you saying that mssql-cli implements intellisense better for Azure SQL DB?

@pensivebrian @MrMeemus

**Comment by anthonydresser** [comment]
Yes, against azure, our SMO models are very slow to be built and that it currently our only method of providing intellisense. The mssql-cli is much quicker since it builds a simpler model.
